catalog/app/components/Preview/renderers/Perspective/Perspective.tsx

```diff
@@ -57,28 +73,42 @@ function ParquetMeta({
     )
 
   return (
-    <div className={cx(classes.root, className)} {...props}>
-      <table className={classes.meta}>
-        <tbody>
-          {renderMeta('Created by:', createdBy, (c: string) => (
-            <span className={classes.mono}>{c}</span>
-          ))}
-          {renderMeta('Format version:', formatVersion, (v: string) => (
-            <span className={classes.mono}>{v}</span>
-          ))}
-          {renderMeta('# row groups:', numRowGroups)}
-          {renderMeta('Serialized size:', serializedSize)}
-          {renderMeta('Shape:', shape, ({ rows, columns }) => (
-            <span>
-              {rows} rows &times; {columns} columns
-            </span>
-          ))}
-          {renderMeta('Schema:', schema, (s: { names: string[] }) => (
-            /* @ts-expect-error */
-            <JsonDisplay value={s} />
-          ))}
-        </tbody>
-      </table>
+    <div className={className} {...props}>
+      <M.Typography className={classes.header} onClick={toggleShow}>
+        <M.Icon
+          className={cx(classes.headerIcon, { [classes.headerIconExpanded]: show })}
+        >
+          expand_more
+        </M.Icon>
+        Parquet metadata
+      </M.Typography>
+      <TransitionGroup>
+        {show && (
```

Comment:
do we actually need this check? `<M.Collapse in={false}>` will hide it contents anyways, no? and it also seems that `TransitionGroup` is not required in this case, or am i missing smth? (probably we should keep one of these -- either transition group with `show && ...` stuff or `<M.Collapse in={show}>`)

Reply:
> M.Collapse reserved space for invisible table.

oh that makes sense actually